quick-start-with-tidb.md

```diff
@@ -6,510 +6,532 @@ aliases: ['/docs/dev/quick-start-with-tidb/','/docs/dev/test-deployment-using-do
+- MySQL compatible
+- Horizontally scalable
+- Real-time HTAP (Hybrid Transactional and Analytical Processing)
+- Cloud-native architecture
 
 > **Note:**
 >
-> The deployment method provided in this guide is **ONLY FOR** quick start, **NOT FOR** production or comprehensive functionality and stability testing.
->
-> - To deploy a self-hosted production cluster, see the [production installation guide](/production-deployment-using-tiup.md).
-> - To deploy TiDB on Kubernetes, see [Get Started with TiDB on Kubernetes](https://docs.pingcap.com/tidb-in-kubernetes/stable/get-started).
-> - To manage TiDB in the cloud, see [TiDB Cloud Quick Start](https://docs.pingcap.com/tidbcloud/tidb-cloud-quickstart).
+> - This deployment method is for development and testing only.
+> - For the fastest way to get started, try [TiDB Cloud Serverless](https://tidbcloud.com/free-trial) - our fully managed service with a free tier.
+> - For production deployments, see [Deploy a TiDB Cluster Using TiUP](https://docs.pingcap.com/tidb/stable/production-deployment-using-tiup) or [Deploy a TiDB Cluster on Kubernetes](https://docs.pingcap.com/tidb/stable/tidb-in-kubernetes).
+
+## Set up your environment
+
+First, install TiUP, the package manager for TiDB:
 
-## Deploy a local test cluster
+```shell
+curl --proto '=https' --tlsv1.2 -sSf https://tiup-mirrors.pingcap.com/install.sh | sh
+```
 
-- Scenario: Quickly deploy a local TiDB cluster for testing using a single macOS or Linux server. By deploying such a cluster, you can learn the basic architecture of TiDB and the operation of its components, such as TiDB, TiKV, PD, and the monitoring components.
+The install script added `tiup` to your shell path. Open a new terminal or reload the shell profile:
 
 <SimpleTab>
 <div label="macOS">
 
-As a distributed system, a basic TiDB test cluster usually consists of 2 TiDB instances, 3 TiKV instances, 3 PD instances, and optional TiFlash instances. With TiUP Playground, you can quickly build the test cluster by following these steps:
+For macOS:
 
-1. Download and install TiUP:
+```shell
+source ~/.zshrc
+```
 
-    {{< copyable "shell-regular" >}}
+</div>
+<div label="Linux">
 
-    ```shell
-    curl --proto '=https' --tlsv1.2 -sSf https://tiup-mirrors.pingcap.com/install.sh | sh
-    ```
+For Linux:
 
-    If the following message is displayed, you have successfully installed TiUP:
-
-    ```log
-    Successfully set mirror to https://tiup-mirrors.pingcap.com
-    Detected shell: zsh
-    Shell profile:  /Users/user/.zshrc
-    /Users/user/.zshrc has been modified to add tiup to PATH
-    open a new terminal or source /Users/user/.zshrc to use it
-    Installed path: /Users/user/.tiup/bin/tiup
-    ===============================================
-    Have a try:     tiup playground
-    ===============================================
-    ```
+```shell
+source ~/.bash_profile
```

Comment:
> source ~/.bash_profile

The current description is too specific and may mislead users, as shell profile paths can vary across different Linux distributions and shell types. I recommend keeping the original, more flexible instructions:

https://github.com/pingcap/docs/blob/063e85c4ffad4cb3d0592698ca1eca9a16e41955/quick-start-with-tidb.md?plain=1#L59-L67

For example, in the installation output shown below, the shell profile is `/home/codespace/.bashrc`:

```bash
Successfully set mirror to https://tiup-mirrors.pingcap.com
Detected shell: bash
Shell profile:  /home/codespace/.bashrc
/home/codespace/.bashrc has been modified to add tiup to PATH
open a new terminal or source /home/codespace/.bashrc to use it
Installed path: /home/codespace/.tiup/bin/tiup
===============================================
Have a try:     tiup playground
===============================================
```

> open a new terminal or source /home/codespace/.bashrc to use it